src/magicgui/signature.py

```diff
@@ -54,8 +54,12 @@ def make_annotated(annotation: Any = Any, options: dict | None = None) -> Any:
     _options = (options or {}).copy()
 
     if get_origin(annotation) is Annotated:
-        annotation, anno_options = get_args(annotation)
-        _options.update(anno_options)
+        # nested Annotated type results in multiple dicts.
+        annotation, *anno_options = get_args(annotation)
+        for opt in anno_options:
+            if not isinstance(opt, dict):
```

Comment:
hmm... was I doing this when it worked before?  I'm not sure I want to prevent people from using Annotated for any other reason.

Reply:
Eh, I guess I'm already preventing it 😂 let's go with it!